lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -387,7 +388,7 @@ then singles, etc..) and then by name.
 
 sub appears_on
```

Comment:
I know it was like this before this PR, but it'd be a good idea to return a hash ref from here so we don't have to make a copy every time it's called.

Also, if you change the query to add the date sorting as suggested, you can probably remove the `recording` join and just use `track.recording` in place of `recording.id`.